audio/audio_driver.c

```diff
@@ -559,6 +561,9 @@ void audio_driver_set_nonblocking_state(bool enable)
  **/
 static void audio_driver_flush(const int16_t *data, size_t samples)
 {
+   if (audio_suspended)
```

Comment:
I think this is the only part where we should not immediately early return.

This is because in RetroArch, we can still do audio mixing of several audio streams which are also processed inside audio_driver_flush.

Try to see if getting rid of this early return still allows everything to work as normal instead.

Reply:
Fine to get rid of that, the function audio_driver_flush is never reached from the main callbacks when suspended.